packages/webapp/components/footer/FooterNavBarV1.tsx

```diff
@@ -58,7 +58,7 @@ const Tab = ({ tab, isActive }: TabProps) => {
   return (
     <FooterNavBarItem
       isActive={isActive}
-      className="flex h-full flex-col items-center justify-center"
+      className="flex h-full flex-col items-center justify-center py-2"
```

Comment:
Why do we need the `py-2` here?

Reply:
Let's make sure there's no conflict with my changes here: https://github.com/dailydotdev/apps/pull/2950 :)

Reply:
Looks good. Thank you :)